.gitignore

```diff
@@ -14,3 +14,4 @@ Source/*.VC.opendb
 Source/*.VC.db
 Source/.vs/
 Source/.vscode/
+/.vs/
```

Comment:
I think this is not necessary, I usually always open the source folder and not the entire thing

Reply:
Fixed in https://github.com/aldelaro5/Dolphin-memory-engine/pull/52/commits/f2d8ea6314fed8dfc59a79e94d281f9069ba69a2 and https://github.com/aldelaro5/Dolphin-memory-engine/pull/52/commits/d8da0b56dce8132f61f741511862868bf0cf4b06